easybuild/easyconfigs/h/HEASoft/HEASoft-6.27.2-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,89 @@
+name = 'HEASoft'
+version = '6.27.2'
+
+
+dependencies = [
+    ( 'CFITSIO', '3.48' ),
+    ( 'Perl', '5.30.2' ),
+]
+
+easyblock = 'ConfigureMake'
+
+homepage = ''
+description = """"""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://www.isdc.unige.ch/~savchenk/cache/heasoft/',
+    'https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft%(version)s/',
+]
+sources = ['heasoft-%(version)ssrc_no_xspec_modeldata.tar.gz']
+patches = []
+
+preconfigopts = "cd BUILD_DIR/ &&"
+prebuildopts = "cd BUILD_DIR/ &&"
+
+# build fails with parallel
+parallel = False
+
+preinstallopts = "cd BUILD_DIR/ &&"
+
+# note that "x86_64-pc-linux-gnu-libc2.17" is hardcoded, since it is derived from toolchain and platform
```

Comment:
So, this installpath thing is definfiely the big issue here. First, I think we should try to tackle this at the install step instead, getting rid of this extra directory. This will just make the rest of the easyconfig muuuuch cleaner. No need to specify PATH and LD_LIBRARY_PATH since they will just work. All the modextravars will be much simpler.

Dealing with the extra path can possibly be done with help of a preinstall/postinstall command, or maybe with a small patch to the makefiles that adds this path (perfectly fine to have a EB-specific patch to do things like this if the configure system of this software doesn't allow for this choice.

Reply:
Ok, I made a patch. Much cleaner indeed!